sigh/lib/sigh/runner.rb

```diff
@@ -105,10 +112,13 @@ def create_profile!
         end
       end
 
-      UI.important "Creating new provisioning profile for '#{Sigh.config[:app_identifier]}' with name '#{name}'"
+      UI.important "Creating new provisioning profile for '#{Sigh.config[:app_identifier]}' with name '#{name}' and platform '#{platform}' - '#{sub_platform}'"
```

Comment:
Minor cosmetic suggestion: 
```
... and platform 'ios' - ''
```
might be a little strange as output, in the case where `sub_platform` is `nil`